core/src/test/java/com/scalar/db/storage/cassandra/CassandraAdminTestBase.java

```diff
@@ -44,20 +53,42 @@
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
-public class CassandraAdminTest {
+/**
+ * Abstraction that defines unit tests for the {@link CassandraAdmin}. The class purpose is to be
+ * able to run the {@link CassandraAdmin} unit tests with different values for the {@link
+ * CassandraAdmin}, notably {@link CassandraAdmin#METADATA_KEYSPACE}.
+ */
+public abstract class CassandraAdminTestBase {
```

Comment:
Abstracted the unit test for Cassandra to be able to test with different values for the metadata keyspace.